Gemfile

```diff
@@ -27,6 +27,7 @@ gem "i18n-js", ">= 3.0.0.rc11"
 gem 'impressionist'
 gem 'jbuilder', '~> 2.11'
 gem 'jquery-rails'
+gem 'mimemagic', '~> 0.3.8' # Upgrade to fix failing builds https://github.com/publiclab/plots2/issues/9377
```

Comment:
Any particular reason we are explicitly mentioning the gem here?

Reply:
As opposed to in the Gemfile.lock? I think we could do it there, right? By running bundle upgrade... Or something?

Reply:
This is noted, doing that now. Thanks @daemon1024 and @jywarren

Reply:
Hmm that's odd. Did you change it manually or with bundler? I wonder if this means it's check summing the file and it doesn't match, or something like that? 

```
opt/hostedtoolcache/Ruby/2.6.6/x64/bin/bundle config --local path /home/runner/work/plots2/plots2/vendor/bundle
  /opt/hostedtoolcache/Ruby/2.6.6/x64/bin/bundle config --local deployment true
  Cache key: setup-ruby-bundler-cache-v3-ubuntu-20.04-ruby-2.6.6-Gemfile.lock-e36a053be2e1d30b9efac37ba7c935ab48554011077f914951ee9f3ae94ea723
  Received 201326592 of 236425156 (85.2%), 192.0 MBs/sec
  Received 236425156 of 236425156 (100.0%), 173.0 MBs/sec
  Cache Size: ~225 MB (236425156 B)
  /usr/bin/tar --use-compress-program zstd -d -xf /home/runner/work/_temp/d2f7efd7-7a17-4006-8e4f-60d1e4ae5bdc/cache.tzst -P -C /home/runner/work/plots2/plots2
  Found cache for key: setup-ruby-bundler-cache-v3-ubuntu-20.04-ruby-2.6.6-Gemfile.lock-e36a053be2e1d30b9efac37ba7c935ab48554011077f914951ee9f3ae94ea723
  /opt/hostedtoolcache/Ruby/2.6.6/x64/bin/bundle install --jobs 4
  You are trying to install in deployment mode after changing
  your Gemfile. Run `bundle install` elsewhere and add the
  updated Gemfile.lock to version control.
  
  If this is a development machine, remove the
  /home/runner/work/plots2/plots2/Gemfile freeze
  by running `bundle config unset deployment`.
  
  You have deleted from the Gemfile:
  * mimemagic (~> 0.3.8)
  Took   6.11 seconds
Error: The process '/opt/hostedtoolcache/Ruby/2.6.6/x64/bin/bundle' failed with exit code 16
```

Any idea? I'll restart just in case that helps...

Reply:
Hi Jeff, I changed this manually... I think next time I'll make sure to use the bundler instead of tweaking the gemfile manually. Thanks!